src/hooks/useGitOpsColumns.tsx

```diff
@@ -0,0 +1,57 @@
+import { TableColumn } from "@openshift-console/dynamic-plugin-sdk";
+import { sortable } from "@patternfly/react-table";
+import * as classNames from 'classnames';
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { GitOpsAppGroupData } from "src/components/utils/gitops-types";
+
+const tableColumnClasses = [
+    classNames.default('pf-m-width-20'), // Application name
+    classNames.default('pf-m-width-30'), // Git repository
+    classNames.default('pf-m-hidden', 'pf-m-visible-on-md', 'pf-m-width-20'), // Environments
+    classNames.default('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-30'), // Last deployment
+  ];
+
+const useGitOpsColumns = (hasSyncStatus: boolean) => {
```

Comment:
Could this be like our GitOpsTableHeader.tsx from before?  eg. Why are we putting this under hooks?  (Probably imported from somewhere?)

Reply:
I followed how kubevirt-plugin used the in their `<VirtualizedTable>`, so it works same way as theirs. Otherwise it would be like `columns=headers` in the `<VirtualizedTable>` properties maybe?
```
<VirtualizedTable<GitOpsAppGroupData>
    data={visibleItems || []}
    unfilteredData={appGroups || []}
    loaded={!emptyStateMsg}
    loadError={null}
    columns={columns}
    Row={GitOpsTableRow}
/>

Reply:
As discussed, we will use `<GitOpsColumns>` instead of `useGitOpsColumns` hook.